lms/djangoapps/grades/context.py

```diff
@@ -36,7 +37,7 @@ def grading_context(course_structure):
         the descriptor tree again.
 
     """
-    all_graded_blocks = []
+    count_all_graded_blocks = 0
```

Comment:
**Note to reviewers:** Changed this to a count, since callers of this function never need the entire list of graded problem blocks.